src/coreclr/jit/gentree.cpp

```diff
@@ -15436,6 +15479,19 @@ bool GenTree::DefinesLocal(Compiler* comp, GenTreeLclVarCommon** pLclVarTree, bo
             blkNode = AsOp()->gtOp1->AsBlk();
         }
     }
+    else if (OperIs(GT_CALL))
+    {
+        GenTree* retBufArg = AsCall()->GetLclRetBufArgNode();
+        if (retBufArg == nullptr)
+        {
+            return false;
+        }
+
+        assert(retBufArg->OperIs(GT_ADDR));
+
```

Comment:
```suggestion
```
(And move inside `SetRetBufArg`, which should probably renamed `SetLclRetBufArg` at that point...)